Bottom: 683ed0edab434e4b1d80f69e5819b7c7e417973e
Top:    29a2692341de66114af0a111f5041d68f085dfa6
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2009-02-10 08:50:01 +0000

Refresh of nearly-working

---

diff --git a/src/polygon1.c b/src/polygon1.c
index b096899..eb33eef 100644
--- a/src/polygon1.c
+++ b/src/polygon1.c
@@ -2641,6 +2641,7 @@ poly_Boolean_free (POLYAREA * ai, POLYAREA * bi, POLYAREA ** res, int action)
       M_POLYAREA_separate_isected (&e, res, &holes, &a_isected);
       M_POLYAREA_label_separated (a_isected, b, FALSE);
       M_POLYAREA_update_primary (&e, res, &holes, action, b);
+//      M_POLYAREA_update_primary_old (&e, res, &holes, &a_isected, action);
       M_POLYAREA_Collect_separated (&e, a_isected, res, &holes, action, FALSE);
 #endif
